integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovements.java

```diff
@@ -404,11 +404,15 @@ void testRetryOccursAndErrorFromIntrospectorLoggedInOperator() throws Exception
     testUntil(() -> checkPodLogContainsRegex(createDomainFailedMsgRegex, operatorPodName, opNamespace),
         logger, "{0} is found in Operator log", createDomainFailedMsgRegex);
 
-    // verify that SEVERE and createDomainFailedMsgRegex message found in Operator log
+    // verify that SEVERE and createDomainFailedMsgRegex message found in introspector log
```

Comment:
Here (in findStringInDomainStatusMessage) you are checking the domain object for the message, not the introspector log.

Reply:
findStringInDomainStatusMessage uses the command "kubectl get domain"